tests/widget/modal/widget-modal.login-with-misc-callbacks.test.js

```diff
@@ -21,15 +22,21 @@ test('Modal widget with simple login and misc callbacks', async ({ page }) => {
   await expect(page.getByText('Are you human?')).toBeVisible();
   await clickButton('Yes', '/authenticate'); // <- Self submitting callback
 
-  // Choice
+  // Choice (select)
   const selectEl = page.getByLabel('Are you sure?');
   await selectEl.selectOption('0');
 
-  // Polling Wait
+  // Choice (radio)
+  const radioEl = page.getByLabel('Are you sure?');
+  await radioEl.selectOption('0');
```

Comment:
If we are going to distinguish between these two (select versus radio), I believe this test wouldn't actually catch the difference as they are selected in the same way. The way it is now, this isn't entirely "fail'able". If we narrow the selection by element type, that would help make if fail'able if the component gets the type wrong. I hope that makes sense.

Reply:
This does make sense, however narrowing down by type is trickier than I'd anticipated. The way to do it seems to be by "role" which we do elsewhere in this PR for spinners. Setting it for "select" is fine as that can map to a "menu", however radio buttons (more specifically groups of radio buttons), are trickier. The reason is that there are aria attributes that we do not currently use and expectations of the group of radio inputs being laid out and labelled in a certain way. Some details are under mozilla docs for [radiogroup](https://developer.mozilla.org/en-US/docs/web/accessibility/aria/roles/radiogroup_role). I think I might have to revisit this in a future PR.

Reply:
Let's just add a TODO comment here to figure out how to differentiate between the two in the future.